### PR TITLE
os_volume: optional bootable parameter for volume

### PR DESCRIPTION
**Pull request by adam-nowotny**
##### SUMMARY
Optional "bootable" parameter for volume

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
os_volume

##### ANSIBLE VERSION
```
devel
```

### CODE REVIEW

File: lib/ansible/modules/cloud/openstack/os_volume.py [hunk 1]
```diff
@@ -45,6 +45,9 @@
    volume:
      description:
        - Volume name or id to create from
+   bootable:
+     description:
+       - Bootable flag for volume
```
Comment (bcoca) [inline lib/ansible/modules/cloud/openstack/os_volume.py]: always add 'version_added' when a new option is added to an existing module

File: lib/ansible/modules/cloud/openstack/os_volume.py [hunk 2]
```diff
@@ -45,6 +45,9 @@
...
      version_added: "2.3"
```
Comment (sivel) [inline lib/ansible/modules/cloud/openstack/os_volume.py]: Additionally, looks like where `bootable` was added, it seems to have separated `version_added: "2.3"` from `volume`.

**Review by achinthagunasekara — APPROVED** [review]

File: lib/ansible/modules/cloud/openstack/os_volume.py [hunk 3]
```diff
@@ -46,6 +46,10 @@
      description:
        - Volume name or id to create from
      version_added: "2.3"
+   bootable:
+     description:
+       - Bootable flag for volume
+     version_added: "2.7"
```
Comment (mnaser) [inline lib/ansible/modules/cloud/openstack/os_volume.py]: Can you set this to 2.8?  Sorry for the delay in how long this took.  We'll get this merged soon.

**Review by mnaser — CHANGES_REQUESTED** [review]

**Comment by mnaser** [comment]
needs_revision

File: lib/ansible/modules/cloud/openstack/os_volume.py (hunk 3 above)
Comment (adam-nowotny) [reply lib/ansible/modules/cloud/openstack/os_volume.py]: sure

**Review by mnaser — APPROVED** [review]
yay!

**Comment by mnaser** [comment]
shipit

**Comment by Akasurde** [comment]
@mnaser @emonty Could you please review this ? Thanks.

**Pull request merged**